src/server.cpp

```diff
@@ -468,6 +468,10 @@ CServer::CServer ( const int          iNewMaxNumChan,
        SIGNAL ( SvrRegStatusChanged() ),
        this, SLOT ( OnSvrRegStatusChanged() ) );
 
+    QObject::connect( &JamRecorder,
```

Comment:
You can directly connect a SIGNAL to a SIGNAL, see, e.g., here: https://github.com/corrados/jamulus/blob/master/src/client.cpp#L156
So you do not need the OnRecordingSessionStarted function.

Reply:
Yeah - I tried that.  It wasn't working -- but when I worked out why, I forgot to put this back again.  (I'd had some debugging in the slot handler.)  I'll fix it.